deblur/workflow.py

```diff
@@ -576,6 +577,10 @@ def get_files_for_table(input_dir,
     file_end : string
         the ending of all the fasta files to be added to the table
         (default '.fasta.trim.derep.no_artifacts.msa.deblur.no_chimeras')
+    num_extra_name_chars : int (optional)
+        the number of additional characters to remove from the end of the
+        file name to get the sample name
+        default is len('.fasta')=6
```

Comment:
@amnona can you provide a specific example on where this is useful? You already have the `file_end` to remove the end of the file name (including all extensions) so I'm really confused on why you need to remove that part.